pulpcore/cli/common/context.py

```diff
@@ -71,6 +66,12 @@ class PulpContext:
     It is an abstraction layer for api access and output handling.
     """
 
+    def echo(self, message: str, nl: bool = True, err: bool = False) -> None:
+        print(message)  # we can do better...
+
+    def prompt(self, text: str, hide_input: bool = False) -> Any:
```

Comment:
Should these just be `NotImplementedError`?

Reply:
I dind't want to make it an abstract class.